x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java

```diff
@@ -395,6 +396,7 @@ public void afterBulk(long executionId, BulkRequest request, Throwable failure)
             .setBulkActions(SETTING_BULK_ACTIONS.get(settings))
             .setBulkSize(SETTING_BULK_SIZE.get(settings))
             .setConcurrentRequests(SETTING_BULK_CONCURRENT_REQUESTS.get(settings))
+            .setBackoffPolicy(BackoffPolicy.noBackoff())
```

Comment:
optionally: add a comment here, why this was added? (OTOH the commit should reveal this as well)